database/migration/src/main/resources/changesets/migration/workflow_store/replace_empty_custom_labels.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE WORKFLOW_STORE_ENTRY
+SET CUSTOM_LABELS = "{}"
+WHERE CUSTOM_LABELS = "";
```

Comment:
I think this SQL step might be unnecessary though. As @mcovarr points out, I don't think this matches anything with `NULL`, but it doesn't matter because when you apply the default in the next migration step, everything gets that default applied.

Have you tried running the migration without this `replace_empty_custom_labels` step?